### ISSUE TITLE
advance to more recent (frozen) brian2 commit due to brian2genn

### ISSUE THREAD

**Issue by moritzaugustin** [brian-team/brian2cuda#17]
Use the following versions in accordance with https://github.com/brian-team/brian2genn/issues/25#issuecomment-184168865

brian2: 71f12749e24f1e5eb6c775c331d8ad5060daad95
brian2genn: a00d75961c9eb95fcbee5c05bbf8cc7602c86765
genn: 0cb5825cbc45ce3b2c4fc0c499c6377b8b31f3fd

**Comment by denisalevi** [comment]
The specified brian2 commit is older then the one previously used for benchmarking (the one that the `frozen` branch from `moritzaugustin/brian2` is build on). Therefore the necessary brian2 patches are not yet merged into brian2 at `71f12749e24f1e5eb6c775c331d8ad5060daad95`.
We could either rebase the patches on moritzaugusting/brian2 to aboves commit or use the current version of the frozen branch anyways. I will check if crucial changed happened btwn the two commits.

**Comment by denisalevi** [comment]
I created a new branch at 'moritzaugustin/brian2', called 'frozen2',  which is the above specified brian2 commit ('71f127') plus the patches necessary for brian2cuda to work.

I ran some test from the brian2 test suit with both, the 'frozen' branch and the 'frozen2' branch, using genn2cuda and brian2cuda. Looks like there are some commits between 'frozen' and 'frozen2' which change some test results.

As discussed we stay with the 'frozen' branch for now.
